src/Microsoft.Health.Fhir.Shared.Api/Features/Formatters/HttpExtensions.cs

```diff
@@ -8,10 +8,11 @@
 using Hl7.Fhir.Rest;
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Logging;
+using Microsoft.Health.Fhir.Core.Features;
 
 namespace Microsoft.Health.Fhir.Api.Features.Formatters
 {
-    public static class SummaryTypeExtensions
+    public static class HttpExtensions
```

Comment:
Maybe HttpContextExtensions?

Reply:
Sounds good, fixed in ae3c2db and 9cdb676.